doc/static/js/ncs.js

```diff
@@ -0,0 +1,133 @@
+function NCS () {
+  "use strict";
+
+  // XXX: do not remove the trailing '/'
+  // const NCS_URL_ROOT = "https://developer.nordicsemi.com/nRF_Connect_SDK/doc/";
+  const NCS_URL_ROOT = "http://localhost:8000/";
+  const NCS_URL_SUFFIX = "/nrf/index.html";
+  const IN_DEVELOPMENT_URL = NCS_URL_ROOT + "latest" + NCS_URL_SUFFIX;
+  const VERSIONS_JSON_URL = NCS_URL_ROOT + "latest" + "/versions.json";
+
+  let state = {
+    current_version: undefined,
+    version_data_url: VERSIONS_JSON_URL,
+  };
+
+  /*
+   * Infer the currently running version of the documentation
+   */
+  state.findCurrentVersion = function() {
+    window.NCS.current_version = window.location.pathname.split("/")[1];
+  };
+
+  /*
+   * This updates the list of versions displayed below the current version.
+   */
+  state.updateVersionDropDown = function() {
+    const current_version = window.NCS.current_version;
+    const versions = window.NCS.versions.VERSIONS;
+
+    let links = '';
+    $.each(versions, function(_, v) {
+      if (v !== current_version) {
+        links += "<li><a href=" + NCS_URL_ROOT + v + NCS_URL_SUFFIX + ">" +
+                  v + "</a></li>";
+      }
+    });
+
+    $("div.version").append("<span class='fa fa-caret-down dropit'></span>" +
+      "<ul class='dropdown'>" + links + "</ul>");
+
+    // hide version dropdown by default & toggle
+    $(".dropdown").hide();
+    $(".dropit").click(function(){
+      $(this).next(".dropdown").toggle();
+    });
+  };
+
+  /*
+   * Display a message at the top of the page to inform the user that the
+   * version currently being browsed is not the latest.
+   */
+  state.showVersion = function() {
+    const VERSIONS = window.NCS.versions.VERSIONS;
+    const latest_release_url = NCS_URL_ROOT + VERSIONS[1] + NCS_URL_SUFFIX;
+
+    const OLD_RELEASE_MSG =
+      "You are looking at an older version of the documentation. You might " +
+      "want to switch to the documentation for the <a href='" +
+      latest_release_url + "'>" + VERSIONS[1] + "</a> release or the " +
+      "<a href='" + IN_DEVELOPMENT_URL + "'>current state of development</a>.";
+
+    const LATEST_RELEASE_MSG =
+      "You are looking at the documentation for the latest release.";
```

Comment:
```suggestion
      "You are looking at the documentation for the last official release.";
```
You're not the only one who thinks "latest" is confusing. ;)

Reply:
Done.